src/services/users/users.hooks.js

```
@@ -53,11 +53,16 @@ module.exports = function (app) {
           }
         ),
         iff(
-          hook => (hook.data && hook.data.email && hook.data.password),
-          sendEmailCode()
+          hook => (hook.data && hook.data.email && hook.data.newEmail && hook.data.password),
```

Are u sure we need both fields - `email` and `newEmail`? I think from UI we just send one email (which is the new one).

Also, the model `User` does not have `newEmail`. I think this should throw an error if you try to patch a user with non-declared field. Did you test this?